code/modules/research/xenobiology/crossbreeding/_products.dm

```diff
@@ -0,0 +1,633 @@
+	desc = "Bluespace technology packed into a roughly egg-shaped device, used to store nonhuman creatures. Can't catch them all, though - it only fits one."
+	w_class = WEIGHT_CLASS_SMALL
+	icon = 'icons/obj/slimecrossing.dmi'
+	icon_state = "capturedevice"
+
+/obj/item/capturedevice/attack(mob/living/M, mob/user)
+	if(contents.len)
+		to_chat(user, "<span class='warning'>The device already has something inside.</span>")
+	if(!isanimal(M))
+		to_chat(user, "<span class='warning'>The capture device only works on simple creatures.</span>")
+		return
+	if(M.mind)
+		to_chat(user, "<span class='notice'>You offer the device to [M].</span>")
+		if(alert(M, "Would you like to enter [user]'s capture device?", "Yes", "No") == "Yes")
+			if(user.canUseTopic(src, BE_CLOSE) && user.canUseTopic(M, BE_CLOSE))
+				to_chat(user, "<span class='notice'>You store [M] in the capture device.</span>")
+				to_chat(M, "<span class='notice'>The world warps around you, and you're suddenly in an endless void, with a window to the outside floating in front of you.</span>")
+				store(M, user)
+			else
+				to_chat(user, "<span class='warning'>You were too far away from [M].</span>")
+				to_chat(M, "<span class='warning'>You were too far away from [user].</span>")
+		else
+			to_chat(user, "<span class='warning'>[M] refused to enter the device.</span>")
+			return
+	else
+		if(istype(M, /mob/living/simple_animal/hostile) && !("neutral" in M.faction))
+			to_chat(user, "<span class='warning'>This creature is too aggressive to capture.</span>")
+			return
+	to_chat(user, "<span class='notice'>You store [M] in the capture device.</span>")
+	store(M)
+
+/obj/item/capturedevice/attack_self(mob/user)
+	if(contents.len)
+		to_chat(user, "<span class='notice'>You open the capture device!</span>")
+		release()
+	else
+		to_chat(user, "<span class='warning'>The device is empty...</span>")
+
+/obj/item/capturedevice/proc/store(var/mob/living/M)
+	M.forceMove(src)
+
+/obj/item/capturedevice/proc/release()
+	for(var/atom/movable/M in contents)
+		M.forceMove(get_turf(loc))
+
+/obj/item/clothing/head/peaceflower
+	name = "heroine bud"
+	desc = "You feel at peace. How could you ever want to feel any different?"
+	icon = 'icons/obj/slimecrossing.dmi'
+	icon_state = "peaceflower"
+	item_state = "peaceflower"
+
+/obj/item/clothing/head/peaceflower/equipped(mob/living/carbon/human/user, slot)
+	. = ..()
+	if(slot == SLOT_HEAD)
+		user.add_trait(TRAIT_PACIFISM, "peaceflower_[REF(src)]")
+
+/obj/item/clothing/head/peaceflower/dropped(mob/living/carbon/human/user)
+	..()
+	user.remove_trait(TRAIT_PACIFISM, "peaceflower_[REF(src)]")
+
+/obj/item/clothing/head/peaceflower/attack_hand(mob/user)
+	if(iscarbon(user))
+		var/mob/living/carbon/C = user
+		if(src == C.head)
+			to_chat(user, "<span class='warning'>You feel at peace. <b style='color:pink'>Why would you want anything else?</b></span>")
+			return
+	return ..()
+
+/obj/item/clothing/suit/armor/heavy/adamantine
```

Comment:
This applies to the whole file, split things up some more. Clothing in the clothing module etc

Reply:
Hegh, that's gonna be a lot of reorganizing.

Reply:
I think I'll end up doing a separate PR to reorganize everything into specific modules, if that's okay with you? It really will be a *ton* of reorganizing, and I want to get it out of the way before continuing with anything else.

Reply:
Yeah that's a no go. You need to think about the thing you're coding and put it into the appropriate place.